### PR TITLE
chore: added prettier import order sorting to the monorepo

### PR DESCRIPTION
**Pull request by FinnDore**
Closes #<issue> 

## ✅ Checklist

- [x] I have followed every step in the [contributing guide](https://github.com/t3-oss/create-t3-app/blob/main/CONTRIBUTING.md) (updated 2022-10-06).
- [x] The PR title follows the convention we established [conventional-commit](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] I performed a functional test on my final commit

---

## Changelog

In #1392 we may be adding prettier import sorting to generated cta apps. this pr adds the same but to the monorepo instead.

looking over the format changes i think they all look fine. i couldn't find any of the problem ones that we don't in #1392 

---

## Screenshots

no lol

💯

### CODE REVIEW

**Comment by FinnDore** [comment]
Wondering if i should revert the formats on the template files... probbaly could be handled in #1413. then again that would cause format linting to fail...

File: cli/template/extras/src/server/api/trpc/base.ts
Comment (juliusmarminge) [inline cli/template/extras/src/server/api/trpc/base.ts:1]: god damnit... don't want to change these files...

File: cli/src/utils/renderVersionWarning.ts [hunk 1]
```diff
@@ -1,6 +1,14 @@
-import { execSync } from "child_process";
 import { getVersion } from "./getT3Version.js";
 import { logger } from "./logger.js";
+import { execSync } from "child_process";
```
Comment (FinnDore) [inline cli/src/utils/renderVersionWarning.ts:3]: Looks like its done it again

Comment (juliusmarminge) [reply cli/src/utils/renderVersionWarning.ts:3]: what's wrong here?

Comment (FinnDore) [reply cli/src/utils/renderVersionWarning.ts:3]: https://github.com/t3-oss/create-t3-app/pull/1392#discussion_r1186910626
its the line bellow. the comment and import were further down before.

Comment (juliusmarminge) [reply cli/src/utils/renderVersionWarning.ts:3]: oh right

**Pull request merged**